frontend/about.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="author" content="Team HYF" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="./styles/css/about.css" />
+    <title>BeHelp</title>
+  </head>
+  <body>
+    <section class="home__about">
+      <div class="home__about-card">
+        <div class="home__about-container">
+          <div class="home__about-header-div">
+            <h1 class="home__about-h1">BeHelp</h1>
+            <h2 class="home__about-h2">WHO WE ARE?</h2>
+          </div>
```

Comment:
Who we are? or Who are we? 😄

Reply:
and not sure why this is also merged to main?

Reply:
I will change to 'Who are we'